webpack/InsightsCloudSync/Components/InsightsTable/InsightsTableActions.js

```diff
@@ -29,12 +27,7 @@ export const fetchInsights = (queryParams = {}) => (dispatch, getState) => {
     select_all: isSelectAll,
   });
 
-  dispatch(
-    push({
-      pathname: INSIGHTS_PATH,
-      search: uri.search(),
-    })
-  );
+  dispatch(push({ search: uri.search() }));
```

Comment:
for reusing the page both in a regular page and also in the host details, I would like to avoid all the conditional checks for hashRouter vs regular router, so yesterday on the UX interest group we discussed about it and today I opened a PR in core which will enable using the search params while on a tab, but when switching tabs - the search params will get cleared: https://github.com/theforeman/foreman/pull/8891

Reply:
also made it work when our component unmounts:

```js
useEffect(
    () => () =>
      dispatch(push({ search: null, hash: hashHistory.location.pathname })),
    [dispatch, hashHistory.location.pathname]
  );
```

but it's still a question if plugins should maintain it, or should the tabs wrapper in core?